docs/content/direct/core-chart.md

```diff
@@ -186,6 +198,9 @@ helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart
   --set-json WDSes='[{"name":"wds1"}]'
 ```
 
+## Step-by-Step Installation Example
+For a detailed step-by-step installation guide with expected outputs, see [Step-by-Step Installation Guide](core-chart-step-by-step-installation.md).
+
 The core chart also supports the use of a pre-existing cluster (or any space, really) as an ITS. A specific application is to connect to existing OCM clusters. As an example, create a first local kind cluster with OCM installed in it:

```

Comment:
This subject --- using a pre-existing cluster as an ITS --- is its own topic, needs its own section heading.